lib/file-system.js

```diff
@@ -7,6 +7,15 @@ exports.exists = function(path) {
   return new Promise((resolve, reject) => fs.exists(path, resolve));
 };
 
+exports.stat = function(path) {
```

Comment:
Actually, `exists` is going away, and stat is one of the replacements ;).
Would a comment suffice here, or should I go all the way and include the logger to drop it on the console?